### ISSUE TITLE
Output only one breadcrumbs on mobile resolution

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/owncloud-design-system#809]
On mobile resolutions, there are duplicate breadcrumbs in the drop. This leads to issues with e.g. tests. There should be only one breadcrumbs.

**Comment by pascalwengerter** [comment]
Is this still the case @LukasHirt ?

**Comment by LukasHirt** [comment]
Yes, it still is. Sorry, the desc is not really clear 🙈 It's not about the breadcrumb items that they'd be displayed twice but there are two breadcrumbs. Instead of having the list only for desktop and drop only for smaller resolution, there are always both and one is "only" hidden. That's fine for looking at the UI but not for the tests which still recognise both of them.

![image](https://user-images.githubusercontent.com/25989331/112444606-8697e880-8d4e-11eb-836a-daaf62365840.png)
